spec/datetimeformat.html

```diff
@@ -303,6 +303,13 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"relatedYear"`, then
+            1. Let _v_ be _tm_.[[relatedYear]].
+            1. Let _fv_ be FormatNumber(_nf_, _v_).
+            1. Add new part record { [[Type]]: `"relatedYear"`, [[Value]]: _fv_ } as a new element of the list _result_.
```

Comment:
if we are using kebab-case, should this be "related-year" instead?

Reply:
Those are option keys (property names) but not the enumeration values. So camelCase is fine here. I had the same question and reviewed this part yesterday.

Reply:
the key to quickly differentiate here what is a property name or a value, is distinguishing `_p_` (property name) and `_f_` (values)